### ISSUE TITLE
powershell with old version like 5.1.19 can't connect to cluster successfully

### ISSUE THREAD

**Issue by kyr1eeee** [kubenetworks/kubevpn#116]
My situation is:
My os is win 10 with powershell of 5.1.19. 
When run command:
`kubevpn proxy deployment/xx --headers a=1`
the process stopped at  `output, err := cmd.CombinedOutput()` below: (no error and no continue)
```golang
func addNicSuffixSearchList(search []string) error {
	cmd := exec.Command("PowerShell", []string{
		"Set-DnsClientGlobalSetting",
		"-SuffixSearchList",
		fmt.Sprintf("@(\"%s\", \"%s\", \"%s\")", search[0], search[1], search[2]),
	}...)
	output, err := cmd.CombinedOutput()
	log.Debugln(cmd.Args)
	if err != nil {
		log.Warnf("error while set dns suffix search list, err: %v, output: %s, command: %v", err, string(output), cmd.Args)
	}
	return err
}
```
I took a few hours to find out this issue and then i remove the calling for addNicSuffixSearchList. ALL WORKS FINE.
```golang
func (c *Config) SetupDNS() error {
	clientConfig := c.Config
	tunName := c.TunName

	tun, err := net.InterfaceByName(tunName)
	if err != nil {
		return err
	}
	luid, err := winipcfg.LUIDFromIndex(uint32(tun.Index))
	if err != nil {
		return err
	}
	var servers []netip.Addr
	for _, s := range clientConfig.Servers {
		var addr netip.Addr
		addr, err = netip.ParseAddr(s)
		if err != nil {
			log.Errorf("parse %s failed: %s", s, err)
			return err
		}
		servers = append(servers, addr)
	}
	err = luid.SetDNS(windows.AF_INET, servers, clientConfig.Search)
	if err != nil {
		log.Errorf("set DNS failed: %s", err)
		return err
	}
	//_ = updateNicMetric(tunName)
	//_ = addNicSuffixSearchList(clientConfig.Search)
	return nil
}

```

I tried to update my powershell to be latest(7.4.0) and tested again.
`kubevpn proxy deployment/xx --headers a=1`
the result is:
```yaml
connectting to cluster
start to connect
got cidr from cache
get cidr successfully
update ref count successfully
traffic manager already exist, reuse it
port forward ready
tunnel connected
adding route...
error while set dns suffix search list, err: exit status 64, output: The argument 'Set-DnsClientGlobalSetting' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Usage: pwsh[.exe] [-Login] [[-File] <filePath> [args]]
                  [-Command { - | <script-block> [-args <arg-array>]
                                | <string> [<CommandParameters>] } ]
                  [-CommandWithArgs <string> [<CommandParameters>]
                  [-ConfigurationName <string>] [-ConfigurationFile <filePath>]
                  [-CustomPipeName <string>] [-EncodedCommand <Base64EncodedCommand>]
                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]
                  [-Interactive] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
                  [-NoProfileLoadTime] [-OutputFormat {Text | XML}]
                  [-SettingsFile <filePath>] [-SSHServerMode] [-STA]
                  [-Version] [-WindowStyle <style>]
                  [-WorkingDirectory <directoryPath>]

       pwsh[.exe] -h | -Help | -? | /?

PowerShell Online Help https://aka.ms/powershell-docs

All parameters are case-insensitive.
, command: [PowerShell Set-DnsClientGlobalSetting -SuffixSearchList @("default.svc.cluster.local", "svc.cluster.local", "cluster.local")]
dns service ok
start to create remote inbound pod for deployment/test-c3004
patch workload default/deployment/test-c3004 with sidecar
rollout status for deployment/test-c3004
Waiting for deployment "test-c3004" rollout to finish: 0 out of 1 new replicas have been updated...
Waiting for deployment "test-c3004" rollout to finish: 1 old replicas are pending termination...
Waiting for deployment "test-c3004" rollout to finish: 1 old replicas are pending termination...
deployment "test-c3004" successfully rolled out
rollout status for deployment/test-c3004 successfully
create remote inbound pod for deployment/test-c3004 successfully
```
Though error occurs when addSuffixSearchList, but the process will continue, and the "proxy" still works.
i wonder is there any solution about this issue?

**Comment by wencaiwulue** [comment]
Yes, i checked the code, this line `addNicSuffixSearchList(clientConfig.Search)` can removed, because of `err = luid.SetDNS(windows.AF_INET, servers, clientConfig.Search)` this method already set dns SearchList, this line can be removed.

**Comment by wencaiwulue** [comment]
I fixed this issue，you can download the latest version v
2.2.1，you can use command `kubevpn upgrade` to upgrade it. Ps: i redo released v2.2.1，so version is the same，but commitid is not same
